weblate/trans/tasks.py

```diff
@@ -685,13 +685,27 @@ def cleanup_project_backup_download() -> None:
 @app.task(trail=False)
 def detect_completed_translation(change_id: int, old_translated: int) -> None:
     change = Change.objects.get(pk=change_id)
-
     translated = change.translation.stats.translated
     if old_translated < translated and translated == change.translation.stats.all:
-        change.translation.change_set.create(
+        change.translation.component.change_set.create(
```

Comment:
I don't see a reason to relocate this to a component with language code in details. It will make it disappear from translation changes.

```suggestion
        change.translation.change_set.create(
```